### ISSUE TITLE
html content does not include summary, whereas md content does

### ISSUE THREAD

**Issue by Dieterbe** [gohugoio/hugo#6686]
### What version of Hugo are you using (`hugo version`)?
current git build (v0.62.0-3-gaa4ccb8a)
<pre>
$ hugo version
Hugo Static Site Generator v0.63.0-DEV linux/amd64 BuildDate: unknown
</pre>

### Does this issue reproduce with the latest release?
Yes, tested with 0.62.0 official release as well.

can easily be reproduced like so:

```
mkdir hugo-test
cd hugo-test
hugo new site quickstart && cd quickstart
mkdir themes && cd themes && git clone https://github.com/yanlinlin82/simple-style.git && cd ..
echo 'theme = "simple-style"' >> config.toml
mkdir content/post
```
```
cat <<EOF > content/post/mdpost.md
---
title: "Mdpost"
date: 2019-12-30T15:21:57+01:00
draft: false
---
summary
<!--more-->
content
EOF
```
```
cat <<EOF > content/post/htmlpost.html
---
title: "Htmlpost"
date: 2019-12-30T15:21:15+01:00
draft: false
---
summary
<!--more-->
content
EOF
```
```
hugo server
```

when you load the html post and the md post in your browser, you can see that the html post does not include the summary, but the markdown does.

**Comment by yanlinlin82** [comment]
Thank you for reporting this. I will check if this is a problem in my theme ('simple-style').

**Comment by Dieterbe** [comment]
no, the problem is the `{{.Content}}` variable, sometimes it contains the summary part of the content, sometimes it only contains the part of the content after the summary divider.

**Comment by yanlinlin82** [comment]
I see. thanks!

**Comment by jmooring** [comment]
Please re-open.

**Comment by ptgott** [comment]
Just a note that if we do want to change this behavior, the place to look is in [this `if` block](https://github.com/gohugoio/hugo/blob/3d5dbdcb1a11b059fc2f93ed6fadb9009bf72673/hugolib/page__per_output.go#L163) or, if #9423 is merged, in [this function](https://github.com/gohugoio/hugo/blob/3d5dbdcb1a11b059fc2f93ed6fadb9009bf72673/hugolib/page__per_output.go#L466).

As it stands now, there is an integration test assertion that HTML content [must not](https://github.com/gohugoio/hugo/blob/2655739940d8148ef374248e867b44d87cd63edf/hugolib/page_test.go#L1334-L1339) include the main content, so the solution might be to document this discrepancy.

**Comment by jmooring** [comment]
This was resolved in v0.134.0.
